packages/contracts/src/systems/NoncombatSkillSystem.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { System } from "@latticexyz/world/src/System.sol";
+
+//import { LibSkill } from "../skill/LibSkill.sol";
+
+contract NoncombatSkillSystem is System {
+  error NoncombatSkillSystem__Asd();
+
+  function execute(bytes32 cycleEntity, bytes32 skillEntity) public override {
```

Comment:
`execute` is a leftover from my misunderstandings of MUD1, and it won't work with MUD2 (all systems' methods exist in one namespace, and can't have the same name)
we should have descriptive method names, like `useSkill`